src/components/container/HeartEssentialEight/HeartEssentialEight.tsx

```diff
@@ -0,0 +1,215 @@
+import MyDataHelps from "@careevolution/mydatahelps-js";
+import { useInitializeView } from "../../../helpers";
+import { Action, LoadingIndicator, ProgressRing, Card } from "../../presentational";
+import React, { useState } from "react";
+import { previewHeartEssentialData } from "./HeartEssentialEight.previewdata";
+import "./HeartEssentialEight.css";
+import iconCholesterol from "./icons/controlcholesterol.svg";
+import iconEatWell from "./icons/eatwell.svg";
+import iconGetActive from "./icons/getactive.svg";
+import iconLoseWeight from "./icons/loseweight.svg";
+import iconManageBloodPressure from "./icons/managebloodpressure.svg";
+import iconReduceBloodSugar from "./icons/reducebloodsugar.svg";
+import iconSleep from "./icons/sleep.svg";
+import iconStopSmoking from "./icons/stopsmoking.svg";
+
+export type HeartEssentialEightScores =
+  | "Activity"
+  | "BloodPressure"
+  | "BloodSugar"
+  | "BMI"
+  | "Cholesterol"
+  | "Diet"
+  | "Nicotine"
+  | "Sleep";
+
+export interface HeartEssentialEightProps {
+  previewState?: "Default" | "NoData";
+  customFieldDictionary: Record<HeartEssentialEightScores, string> & {
+    [key: string]: string;
+  };
+  innerRef?: React.Ref<HTMLDivElement>;
+}
+
+export default function (props: HeartEssentialEightProps) {
+  const customRingStyle: React.CSSProperties = {
+    backgroundColor: "lightblue",
+    height: "100px",
+    width: "100px",
+  };
+
+  let [score, setScore] = React.useState<number>();
+  let [achievementMap, setAchievementMap] =
+    useState<Record<HeartEssentialEightScores, boolean>>();
+
+  async function initialize() {
```

Comment:
I don't think you need the `async` keyword here.  No calls to this function are awaiting or expecting a promise.